Travel Scheduler/View Controllers/FirstScreenViewController.h

```diff
@@ -27,7 +27,7 @@ NS_ASSUME_NONNULL_BEGIN
 @property (strong, nonatomic) NSString *userSpecifiedPlaceToVisit;
 @property (strong, nonatomic) NSDate *userSpecifiedStartDate;
 @property (strong, nonatomic) NSDate *userSpecifiedEndDate;
-@property (strong, nonatomic) UILabel *headerLabel;
+//@property (strong, nonatomic) UILabel *headerLabel;
```

Comment:
I this commented out or will still be in use for the future. Consider removing it if not in use.